Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1942,6 +1945,17 @@ def lhegensim2018ml(fragment,howMuch):
                             '-n':'10'
                             },steps['RECOHID15']])
 
+steps['REMINIAODHID18']={ '--scenario':'pp',
+                          '--conditions':'auto:run2_data_promptlike_hi',
+                          '-s':'PAT,DQM:@miniAODDQM',
+                          '--datatier':'MINIAOD,DQMIO',
+                          '--eventcontent':'MINIAOD,DQM',
+                          '--era':'Run2_2018_pp_on_AA',
+                          '--data':'',
+                          '--processName':'PAT',
+                          '-n':'10'
```

Comment:
can we do 100 events? miniAOD is supposed to be fast and the output should be small enough to afford more events.
Same comment for the MC case.
This will match what's available for pp